content/concepts/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Concepts"
+weight = 4
+chapter = false
```

Comment:
should we create a list of pages here?

Reply:
thanks, i missed that . check the new changes